tests/bits/step-16.cc

```diff
@@ -1,6 +1,6 @@
 // ---------------------------------------------------------------------
 //
-// Copyright (C) 2005 - 2013 by the deal.II authors
+// Copyright (C) 2005 - 2013, 2015 by the deal.II authors
```

Comment:
The problem is that most of our patches don't update the copyright year. emacs does it for me, but I only sometimes use emacs. Most of my patches are written in eclipse, and eclipse doesn't update the copyright year. So I think that for most files, the current state does not adequately reflect the year anyway. Thus my suggestion to do it ahead of every release for all files.